backend/src/LibExecution/RuntimeTypes.fs

```diff
@@ -400,13 +400,75 @@ and [<RequireQualifiedAccess>] ValueType =
   | Unknown
   | Known of KnownType
 
-/// VTTODO if this is used somewhere, re-evaluate the usage - it feels there's something to be done...
-/// CLEANUP eventually remove this binding
-let valueTypeTODO = ValueType.Unknown
-let valueTypesTODO = []
+[<RequireQualifiedAccess>]
+module ValueType =
+  // some helpers to reduce typing elsewhere
+  let unknown = ValueType.Unknown
+  let unknownTODO = ValueType.Unknown
+  let unknownDbTODO = ValueType.Unknown
+  let uknownTypeArgsTODO = []
```

Comment:
What does uknownTypeArgs mean?

Reply:
I just renamed this binding to `typeArgsTODO` in another PR. I think that's more correct. The intention here is to imply that some type (like Option) needs some typeArgs but we haven't typed them out yet when constructing the Enum/Record - so these placeholders are a call-out to follow up later, filling them in.